openshift/installer
# Bug 2043297: bump RHCOS 4.10 bootimage metadata

These changes will update the RHCOS 4.10 bootimage metadata in the installer.
This change includes fixes for the following BZs:

Bug 2008521 - gcp-hostname service should correct invalid search entries in resolv.conf
Bug 2043296 - Ignition fails when reusing existing statically-keyed LUKS volume
Bug 2043721 - Installer bootstrap hosts using outdated kubelet containing bugs
Bug 2043316 - RHCOS VM fails to boot on Nutanix AOS

This change will also introduce artifacts for for Aliyun, AWS GovCloud regions, and Nutanix.

Changes generated with:

```
$ cosa shell
[coreos-assembler]$ plume cosa2stream --target data/data/coreos/rhcos.json --distro rhcos --no-signatures \
--url https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases aarch64=410.84.202201251203-0 \
ppc64le=410.84.202201251004-0 s390x=410.84.202201251002-0  x86_64=410.84.202201251210-0
```
Verification Steps:

Install a new 4.10 cluster
oc debug node/<node name> -- chroot /host rpm-ostree status
Verify that the deployment version matches the version from this PR
that matches the architecture you are testing on. (i.e. x86_64
should have version 410.84.202201251210-0)
